Bottom: 5c71167b7e56b10865ce13dfb5bf4199543d1267
Top:    083fe104693e36085eaeff25e1c4d2628b62582f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-11 15:31:20 +0000

Avoid clearing pins / vias from polygons... it is already done!



---

diff --git a/src/draw.c b/src/draw.c
index f282b44..fdb33f7 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1239,6 +1239,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1261,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
